### PR TITLE
docs: structure, content & awesome-lint action

### PR DESCRIPTION
**Pull request by marc0olo**
contributing guidelines need to be added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,405 @@
...
+    - https://blog.aeternity.com/why-run-an-ae-node-and-how-to-do-it-8b95a685f683
+- You should ***never*** rely on the hosted infrastructure of aeternity for applications running in production!
+
+## Explorers
+
+### Mainnet
+- [AEknow](https://aeknow.org) - Explorer that also shows mining stats developed by the community.
+    - https://github.com/jdgcs/aeknow
+- [aenalytics](https://aenalytics.org) - Explorer with focus on UX and the aeternity naming system (AENS).
+- [Indaex](https://explorer.aeternity.io) - Default explorer provided by aeternity.
```
Comment (thepiwo) [inline README.md:138]: Indaex is not the correct name, right?

File: README.md [hunk 2]
```diff
@@ -1,69 +1,405 @@
...
-- [AENS Marketplace](https://github.com/u2467/aens-marketplace) - Marketpalce for AENS (Python/Sophia) - from Hack&Play/11.2019 
-- [TypeScript demo](https://github.com/gcofficial/ae-typescript-demo) - Use the JS SDK with TypeScript
+
+### Browser extensions
+- Superhero Wallet - The most advanced aeternity wallet with additional features for Superhero.com.
+    - [Brave / Chrome](https://chrome.google.com/webstore/detail/superhero/mnhmmkepfddpifjkamaligfeemcbhdne?hl=en)
+    - [Firefox](https://addons.mozilla.org/en-US/firefox/addon/superhero-wallet/)
+- [Waellet](https://github.com/waellet/waellet) - Currently not actively developed.
+
+### Cloud & custodial wallets
```
Comment (thepiwo) [inline README.md:159]: what about the chinese web wallet, don't remember the name

File: README.md [hunk 3]
```diff
@@ -1,69 +1,405 @@
...
+- [aepp-sdk-net](https://github.com/block-m3/aepp-sdk-net) - Outdated.
+
+### Smart Contract Development (Sophia language)
+
+###### Compiler
+- [aesophia](https://github.com/aeternity/aesophia) - Stand alone compiler for the Sophia smart contract language.
+- [aesophia_cli](https://github.com/aeternity/aesophia_cli) - The command line client for the Sophia compiler.
+- [aesophia_http](https://github.com/aeternity/aesophia_http) - The http interface to the Sophia compiler.
+
+###### REPL (read–eval–print loop)
```
Comment (thepiwo) [inline README.md:225]: maybe add a link for the hosted repl

File: README.md (hunk 1 above)
Comment (marc0olo) [reply README.md:138]: Good question. There was a branding vote few months ago. I think it should be branded like this then. But agree, currently it isn't called "Indaex". Mhhh 😅

Comment (marc0olo) [reply README.md:138]: https://forum.aeternity.com/t/middleware-new-branding-vote/8186/14

Comment (marc0olo) [reply README.md:138]: But I guess it also wasn't clear if the branding is targeting the backend or the frontend of the middleware. Or even both.

File: README.md (hunk 2 above)
Comment (marc0olo) [reply README.md:159]: Not sure about that one. Could also be added later on. So AEBox is already included in the mobile app section and I don't think there exists a web wallet for it.

If you mean https://aeasy.io -> I added it under development framework as it describes itself to be.

And WeTrue is under the applications list as social aepp.

File: README.md (hunk 1 above)
Comment (marc0olo) [reply README.md]: Renamed to `aeternity explorer`

File: README.md (hunk 3 above)
Comment (marc0olo) [reply README.md:226]: added

**Review by thepiwo — APPROVED** [review]

**Pull request merged**